Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1087,11 +1087,11 @@ void RenderGrid::placeItemsOnGrid(std::optional<LayoutUnit> availableLogicalWidt
     performAutoPlacement();
 
 #if ASSERT_ENABLED
-    for (auto* gridItem = currentGrid().orderIterator().first(); gridItem; gridItem = currentGrid().orderIterator().next()) {
-        if (currentGrid().orderIterator().shouldSkipChild(*gridItem))
+    for (RenderBox& gridItem : childrenOfType<RenderBox>(*this)) {
```

Comment:
I think we try to use auto throughout (newly written) layout code so maybe we should be consistent

Reply:
changed